inputfiles/sherpa/LHC-LO-ZplusJet/Run.dat

```diff
@@ -18,11 +18,12 @@
 (processes){
   Process 93 93 -> 13 -13 93;
   Order (*,2);
+  CKKW sqr(QCUT/E_CMS);
+  Cut_Core 1;
   End process;
 }(processes)
 
 (selector){
   Mass 13 -13 60. 120.;
-  NJetFinder 1 20. 0. 0.7 -1;
```

Comment:
Why is the jet finder not needed anymore? Because of properly defined matching cut?

Reply:
Because of the CKKW QCut matching with only one jet. So this is ~equivalent.